### PR TITLE
Total area available chart

### PR DESCRIPTION
**Pull request by tinashechiraya**
[Screencast from 08-10-2023 16:11:47.webm](https://github.com/kartoza/sawps/assets/70011086/08863baf-116c-474a-8b8d-a6657c670b2f)

Description:
the chart requires a species to be selected in the filters to fetch the total area available for that species
by default it fetches data for the years where data is available(was recorded)
the user can set the years in the filters to get specific results
the colors are dynamically and randomly assigned to the years available and mapped to the corresponding bar on the chart
the chart follows the style from the graph matrix document

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 11 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 10 min

**Comment by tinashechiraya** [comment]
@dimasciput its failing with that build docker Images

File: django_project/frontend/tests/test_metrics.py [hunk 1]
```diff
@@ -54,6 +54,19 @@ def setUp(self):
             5, taxon=self.taxon, user=self.user, property=self.property
         )
 
+        # self.taxon1 = TaxonFactory.create(
```
Comment (dimasciput) [inline django_project/frontend/tests/test_metrics.py:57]: Please remove if it's not needed anymore

**Review by dimasciput — CHANGES_REQUESTED** [review]

File: django_project/frontend/tests/test_metrics.py (hunk 1 above)
Comment (tinashechiraya) [reply django_project/frontend/tests/test_metrics.py]: @dimasciput I have done so

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 16 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 15 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 55 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 11 min

**Pull request merged**